editor/animation_track_editor.cpp

```diff
@@ -2699,7 +2699,11 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 						StringName prop = npath.get_concatenated_subnames();
 						PropertyInfo prop_info;
 						ClassDB::get_property_info(nd->get_class(), prop, &prop_info);
-						bool is_angle = prop_info.type == Variant::FLOAT && prop_info.hint_string.find("radians") != -1;
+						bool is_angle = prop_info.type == Variant::FLOAT && (prop_info.hint_string.find("radians_as_degrees") != -1
+#ifndef DISABLE_DEPRECATED
+																					|| prop_info.hint_string.find("radians") != -1
+#endif
+																			);
```

Comment:
Could do this as:
```suggestion
						
#ifdef DISABLE_DEPRECATED
						bool is_angle = prop_info.type == Variant::FLOAT && prop_info.hint_string.find("radians_as_degrees") != -1;
#else
						bool is_angle = prop_info.type == Variant::FLOAT && prop_info.hint_string.find("radians") != -1;
#endif
```
Instead

Reply:
Yeah, full statement in each block would be better

Reply:
Good, have done and will push when the CI has finished as it touches `object.h`